### PR TITLE
add "normal" RPM and image building

### PR DESCRIPTION
**Pull request by deads2k**
@smarterclayton who should review this sort of thing?

### CODE REVIEW

File: service-serving-cert-signer.spec [hunk 1]
```diff
@@ -49,9 +49,9 @@
 %global golang_version 1.8.1
 %{!?version: %global version 0.0.1}
 %{!?release: %global release 1}
-%global package_name service-serving-cert-signer # PACKAGE_NAME
-%global product_name TODO # PRODUCT_NAME
-%global import_path TODO # GO_PACKAGE
+%global package_name origin-service-serving-cert-signer
```
Comment (smarterclayton) [inline service-serving-cert-signer.spec]: You don't need `origin` in the package name anymore.

File: images/service-serving-cert-signer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#
+# This is the integrated OpenShift OService Serving Cert Signer.  It signs serving certificates for use inside the platform.
+#
+# The standard name for this image is openshift/origin-service-serving-cert-signer
+#
+FROM openshift/origin-base
+
+RUN INSTALL_PKGS="origin-service-serving-cert-signer" && \
```
Comment (smarterclayton) [inline images/service-serving-cert-signer/Dockerfile]: Remove `origin` or use `openshift` for the package name.  I guess we need *some* prefix though...

**Comment by deads2k** [comment]
switched prefix to openshift

**Comment by deads2k** [comment]
typo fixed.  plan to merge on green to push forward.

**Comment by deads2k** [comment]
Alright, we'll continue from here.

**Pull request merged**